### DISCUSSION TITLE
How to disable verbose consoles logging?

### DISCUSSION THREAD

**Discussion by acrabb** [invertase/react-native-firebase#4913] [Q&A]
Hey everyone,

I've noticed the Xcode console gets pretty verbose with logs from `Firebase/Analytics`.
How can I disable so much logging out from Analytics?
If there is no option to do so, is a feature request open for this?

Cheers,
Andre

**Comment by mikehardy** [comment]
If it is from 'Firebase/Analytics' - that is directly from the firebase-ios-sdk then I am not aware of the options personally (they may exist! I sincerely just do not know) but the upstream docs should say? https://firebase.google.com/docs/analytics/get-started?platform=ios
